cloud-gateway/src/main/java/com/registry/cloudgateway/dto/RestResponsePage.java

```diff
@@ -0,0 +1,37 @@
+package com.registry.cloudgateway.dto;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RestResponsePage<T> extends PageImpl<T> {
+
+    private static final long serialVersionUID = 3248189030448292002L;
```

Comment:
I don't think you need this serialVersionUID. Fields like this one are used when you are using Java serialization. This class does not implement Serializable, so I don't think we need it.